config/initializers/csv_safe.rb

```diff
@@ -0,0 +1,12 @@
+require "csv-safe"
+
+class CSVSafe
+  def sanitize_field(field)
+    # we may represent monetary numerics as strings to format them, so we don't want CSVSafe to prefix them
+    if field.nil? || field.is_a?(Numeric) || field.is_a?(String) && Regexp.new(/\A[-\d.,]+\z/).match?(field)
```

Comment:
Because this conditional is quite long, could we extract the third part into a variable?

```ruby
field_is_currency = field.is_a?(String) && Regexp.new(/\A[-\d.,]+\z/).match?(field)
if field.nil? || field.is_a?(Numeric) || field_is_currency
```

I also think the operator precedence is quite confusing here, so extracting into a variable fixes that too!